src/tracer/TraceExecutor.cc

```diff
@@ -58,26 +59,29 @@ executor::ExecuteAction TraceExecutor::onExecuteEvent(
     if (!hasExecved_ && executeEvent.trapped &&
         executeEvent.signal == (SIGTRAP | (PTRACE_EVENT_EXEC << 8))) {
         hasExecved_ = true;
-        for (auto& listener: eventListeners_)
+        for (auto& listener: eventListeners_) {
             listener->onPostExec(event);
+        }
     }
 
     TraceAction action = TraceAction::CONTINUE;
-    for (auto& listener: eventListeners_)
+    for (auto& listener: eventListeners_) {
         action = std::max(action, listener->onTraceEvent(event, tracee));
+    }
 
     if (!executeEvent.exited && !executeEvent.killed && tracee.isAlive()) {
         int64_t signal = executeEvent.signal & 0xff;
-        if (executeEvent.trapped && signal == SIGTRAP)
+        if (executeEvent.trapped && signal == SIGTRAP) {
             signal = 0;
+        }
 
         if (signal > 0) {
             // Since our child is pid 1 we have to kill on delivery on uncaught
             // signal in favour of kernel.
             uint64_t caughtSignals =
                     procfs::readProcFS(traceePid_, procfs::Field::SIG_CGT);
             caughtSignals |= IGNORED_SIGNALS;
-            if (!(caughtSignals & (1 << signal))) {
+            if ((caughtSignals & (1 << signal)) == 0U) {
```

Comment:
IMO wcześniejsze było bardziej czytelne, ale to może kwestia przyzwyczajenia

Reply:
Mhm